docs/wiki/Overview.md

```diff
@@ -29,7 +29,7 @@ Each metric fetcher will be assigned a few number of partitions in the cluster t
 #### Metric Sample Aggregator
 Metric Sample Aggregator is a module that organizes all the metric samples. The metric sample aggregator will put each metric sample into a workload snapshot according to the timestamp of the metric sample. A workload snapshot is the aggregated metric information during a given time window. For example, if the snapshot time window is 1 hour and user configured to keep 168 snapshots. Kafka Cruise Control will store 7 days of hourly workload information.
 
-When queried, the metric sampler aggregator will return all the aggregated workload snapshot back to the LoadMonitor. Users can define a minimum number of metric samples required in each workload snapshot to ensure the workload snapshot has enough metric samples. For example, if user specifies snapshot window to be 1 hour and the sampling interval is 5 minutes, ideally each snapshots should have 12 metric samples. User can define that a snapshot window with less than 10 metric samples is invalid and should not be included into the cluster workload model.
+When queried, the metric sampler aggregator will return all the aggregated workload snapshot back to the LoadMonitor. Users can define a minimum number of metric samples required in each workload snapshot to ensure the workload snapshot has enough metric samples. For example, if user specifies snapshot window to be 1 hour and the sampling interval is 5 minutes, ideally each snapshot should have 12 metric samples. User can define that a snapshot window with less than 10 metric samples is invalid and should not be included into the cluster workload model.
```

Comment:
Here are some other issues that we can fix in this document:
* `workload` -> `load` (i.e. the term `workload` is no longer the preferred way to refer to the `load`. For consistency, can we adopt `load` in this document?).
* `snapshot` -> `window` (i.e. the term `snapshot` was used to be the preferred way to refer to a `window`. To make it more confusing, the documentation also uses the term "time window" to refer to the period from the start to the end of a `window`. Can we update those to improve the clarity?)
* there are also other occurrences of the same issue (e.g. `each of sampling tasks...`, `Each sampling tasks...`)
* `Kafka Cruise Control` -> `Cruise Control`

Reply:
Done! Thanks for your advice.